tests/validator/test_validator.py

```diff
@@ -201,68 +201,6 @@ def test_validator_default_expectation_args__sql(
         )
 
 
-@pytest.mark.big
-def test_columns(
```

Comment:
These are methods that exist on the legacy validator and aren't part of the new one - deleting because the test setup required some complex legacy checkpoints